pymdownx/highlight.py

```diff
@@ -52,7 +52,7 @@
         'Default: True'
     ],
     'guess_lang': [
-        False,
+        0,
```

Comment:
I think what would make sense is to set this to `None`. If `None`, we will assume the default of `False`. Otherwise, we will accept booleans or `block` or `inline`. No other strings should be accepted.